src/cookbooks/looker/intro.md

```diff
@@ -60,7 +60,7 @@ The following video demonstrates this workflow in detail:
 
 If you want to explore Looker more deeply, you can check out:
 
-- ["Looker Connect" training hub](https://connect.looker.com/library): A collection of self-paced video training courses for new users. Full courses are free, but require registration, but course descriptions contain preview videos that are useful on their own.
+- ["BI and Analytics with Looker" training hub](https://www.cloudskillsboost.google/journeys/28): A collection of self-paced video training courses for new users. Full courses are free, but require registration, but course descriptions contain material that is useful on its own.
```

Comment:
I'm unsure about this one. "Looker Connect" doesn't exist anymore, are the linked lab courses the same/similar? cc @scholtzan

Reply:
yeah, those are basically the same